### PR TITLE
feat(query): add PostgreSQL Database Server Log Connections Disabled for ARM

### PR DESCRIPTION
**Pull request by rogeriopeixotocx**
**Proposed Changes**
- adding query 'PostgreSQL Database Server Log Connections Disabled' for ARM

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Review by joaoReigota1 — APPROVED** [review]
LGTM

File: assets/queries/azureResourceManager/postgres_sql_server_log_connections_disabled/query.rego [hunk 1]
```diff
@@ -49,7 +50,22 @@ CxPolicy[result] {
...
+CxPolicy[result] {
+	resource := input.document[i].resources[_]
+	resource.type == "Microsoft.DBforPostgreSQL/servers/configurations"
+	endswith(resource.name, "log_connections")
+	not common_lib.valid_key(resource.properties, "value")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("resources.name={{%s}}", [resource.name]),
+		"issueType": "IncorrectValue",
```
Comment (rafaela-soares) [inline assets/queries/azureResourceManager/postgres_sql_server_log_connections_disabled/query.rego:67]: `"issueType": "MissingAttribute"`

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/azureResourceManager/postgres_sql_server_log_connections_disabled/query.rego [hunk 2]
```diff
@@ -5,37 +5,38 @@ import data.generic.common as common_lib
...
 	childValue.type == "configurations"
-	not endswith(childValue.name, "log_connections")
+	childValue.name == "log_connections"
+	not common_lib.valid_key(childValue.properties, "value")
 
 	result := {
 		"documentId": input.document[i].id,
-		"searchKey": sprintf("%s.name={{%s}}.%s.name=%s", [common_lib.concat_path(parentPath), parentName, common_lib.concat_path(childPath), childValue.name]),
+		"searchKey": sprintf("%s.name={{%s}}.resources.name=log_connections", [common_lib.concat_path(parentPath), parentName]),
 		"issueType": "IncorrectValue",
```
Comment (rafaela-soares) [inline assets/queries/azureResourceManager/postgres_sql_server_log_connections_disabled/query.rego:37]: `"issueType": "MissingAttribute"`

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGTM

**Pull request merged**